spec/index.html

```diff
@@ -795,18 +803,18 @@ <h3>Examples</h3>
 
         <table id="ex-ca-shared-result-e0"
                class="hash-related numbered">
-          <caption>Result for `_:e0`</caption>
+          <caption>Result for `e0`</caption>
           <tbody>
             <tr>
               <td><var>temporary issuer</var> mappings</td>
               <td>
                 <table class="hash-related">
                   <thead><tr>
-                    <th><var>original</var></th>
-                    <th><var>canonical</var></th>
+                    <th><var>blank node</var></th>
```

Comment:
This seems to be the main area of confusion. Perhaps "original identifier" is more clear here?